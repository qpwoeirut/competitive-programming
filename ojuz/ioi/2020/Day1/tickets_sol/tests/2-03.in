df804a0e-e4b1-4b8d-a0f5-83401a72b5a3
10 10 1
384973315 455938459 582063496 695771995 750880989 753228128 817432892 946654411 973780279 988124939
14792264 63790970 111177175 130437146 156520233 478517979 525349601 555140911 894776334 943823210
4278110 36905749 235585558 277143168 344214076 546871979 614274442 801694556 911064916 924639675
240778127 248279928 316029939 451898042 494674698 530813206 576467843 627578480 732035192 954290424
132030457 349974438 412677807 566797287 582322700 598814054 648767042 660425322 928770856 970316227
79813508 158500222 521404189 542054447 686752950 789466562 804133490 832941589 851805435 945643516
39879753 48873008 133274590 443500151 462093748 624565291 628475588 721594155 906344637 943696827
209016129 239580790 280508248 428075429 467469723 497285991 510212646 525742298 691343377 907596493
145709529 150267697 171621120 424043606 628569338 753317814 838615741 891670574 970386185 994082034
250866919 258944076 342511874 450996183 537240884 545591244 606810479 650863590 774636126 991007391
