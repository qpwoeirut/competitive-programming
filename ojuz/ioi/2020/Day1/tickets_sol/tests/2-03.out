131b432f-1f84-4f8f-9d55-41e868a6bc5d
OK
4564307189
-1 -1 -1 -1 -1 -1 -1 -1 -1 0
0 -1 -1 -1 -1 -1 -1 -1 -1 -1
0 -1 -1 -1 -1 -1 -1 -1 -1 -1
-1 -1 -1 -1 -1 -1 -1 -1 -1 0
0 -1 -1 -1 -1 -1 -1 -1 -1 -1
0 -1 -1 -1 -1 -1 -1 -1 -1 -1
0 -1 -1 -1 -1 -1 -1 -1 -1 -1
-1 -1 -1 -1 -1 -1 -1 -1 -1 0
-1 -1 -1 -1 -1 -1 -1 -1 -1 0
-1 -1 -1 -1 -1 -1 -1 -1 -1 0
